data/resources/ui/components-message-entry.ui

```diff
@@ -30,21 +30,16 @@
                 <property name="hexpand">True</property>
                 <property name="child">
                   <object class="GtkTextView" id="text_view">
-                    <property name="top-margin">6</property>
                     <property name="bottom-margin">6</property>
-                    <property name="left-margin">9</property>
-                    <property name="right-margin">9</property>
+                    <property name="top-margin">8</property>
```

Comment:
In that case text looks shifted to top
![image](https://user-images.githubusercontent.com/63008755/202758446-2d7636dd-ea3f-4925-bdc3-b2947eb5e71c.png)

message entry is highter than message bubble (34px vs 31px)
we need to either use diffrent margins or reduce the message entry height
![image](https://user-images.githubusercontent.com/63008755/202758681-6d5a5ce3-5339-469b-aa31-b174ba5d123f.png)

Reply:
> In that case text looks shifted to top

This issue wasn't present before those changes though, what's different about it?

> message entry is highter than message bubble (34px vs 31px)

That's not really a problem imho.

Reply:
>This issue wasn't present before those changes though, what's different about it?

You removed `valign=center`, so that's why it's not aligned anymore.

Reply:
> This issue wasn't present before those changes though, what's different about it?

TextView was aligned to center, i've returned that behavior in last commit